### PR TITLE
KG - Datepicker Modifications

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

**Comment by kyle-glick** [comment]
This is ready to be merged after the deploy to production

**Comment by kyle-glick** [comment]
This branch currently has a weird bug related to bootstrap datetimepicker. I currently have an issue up on the datetimepicker repository and am waiting for a response.

File: Gemfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ gem 'haml-rails'
 gem 'hashie-forbidden_attributes'
 gem 'jquery-rails'
 gem 'jbuilder', '~> 2.0'
-gem 'momentjs-rails', '>= 2.8.1'
+gem 'momentjs-rails', '>= 2.9.0'
```
Comment (wtholt) [inline Gemfile]: Why did you need to update the libraries?

File: app/views/tasks/_form.html.haml [hunk 2]
```diff
@@ -16,17 +16,15 @@
             .form-group
               = f.label "assignee_id", t(:task)[:assignee_name]+t(:actions)[:required], class: "col-sm-4 control-label"
```
Comment (wtholt) [inline app/views/tasks/_form.html.haml]: Cool. New Syntax.

File: db/schema.rb [hunk 3]
```diff
@@ -13,7 +13,6 @@
```
Comment (wtholt) [inline db/schema.rb]: Why is the schema checked in?

**Comment by kyle-glick** [comment]
@wtholt
I had to update the gem because the features requested for this story became available in momentjs 2.9.0

As for the schema, all I did was remove an extra line that I noticed.
